The command nx test should properly run 'go test'

Current behavior:

```
$ nx test api

> nx run api:test
Executing command: go test apps/api/src/main_test.go
# command-line-arguments [command-line-arguments.test]
apps/api/src/main_test.go:8:12: undefined: Hello
FAIL	command-line-arguments [build failed]
FAIL
Failed to execute command: go test apps/api/src/main_test.go
```

--- a/e2e/nx-go-e2e/tests/application.test.ts
+++ b/e2e/nx-go-e2e/tests/application.test.ts
@@ -14,8 +14,9 @@ describe('application e2e', () => {
     const resultBuild = await runNxCommandAsync(`build ${appName}`);
     expect(resultBuild.stdout).toContain(`Executing command: go build`);
 
-    // const resultLint = await runNxCommandAsync(`lint ${plugin}`);
-    // expect(resultLint.stdout).toContain(`Executing command: go fmt`);
+    const resultLint = await runNxCommandAsync(`lint ${appName}`);
+    expect(resultLint.stdout).toContain(`Executing command: go fmt apps/${appName}/**/*.go`);
+    expect(resultLint.stdout).toContain(`apps/`);
 
     const resultServe = await runNxCommandAsync(`serve ${appName}`);
     expect(resultServe.stdout).toContain(`Executing command: go run`);

--- a/e2e/nx-go-e2e/tests/application.test.ts
+++ b/e2e/nx-go-e2e/tests/application.test.ts
@@ -7,21 +7,21 @@ import {
 } from '@nrwl/nx-plugin/testing';
 describe('application e2e', () => {
   it('should create application', async (done) => {
-    const plugin = uniq('app');
+    const appName = uniq('app');
     ensureNxProject('@nx-go/nx-go', 'dist/packages/nx-go');
-    await runNxCommandAsync(`generate @nx-go/nx-go:application ${plugin}`);
+    await runNxCommandAsync(`generate @nx-go/nx-go:application ${appName}`);
 
-    const resultBuild = await runNxCommandAsync(`build ${plugin}`);
+    const resultBuild = await runNxCommandAsync(`build ${appName}`);
     expect(resultBuild.stdout).toContain(`Executing command: go build`);
 
     // const resultLint = await runNxCommandAsync(`lint ${plugin}`);
     // expect(resultLint.stdout).toContain(`Executing command: go fmt`);
 
-    const resultServe = await runNxCommandAsync(`serve ${plugin}`);
+    const resultServe = await runNxCommandAsync(`serve ${appName}`);
     expect(resultServe.stdout).toContain(`Executing command: go run`);
 
-    // const resultTest = await runNxCommandAsync(`test ${plugin}`);
-    // expect(resultTest.stdout).toContain(`Executing command: go test`);
+    const resultTest = await runNxCommandAsync(`test ${appName}`);
+    expect(resultTest.stdout).toContain(`Executing command: go test apps/${appName}/**/*.go`);
 
     done();
   });

--- a/packages/nx-go/src/builders/lint/builder.ts
+++ b/packages/nx-go/src/builders/lint/builder.ts
@@ -9,10 +9,11 @@ export function runBuilder(
   context: BuilderContext
 ): Observable<BuilderOutput> {
   return from(context.getProjectMetadata(context?.target?.project)).pipe(
-    map(() => {
-      const mainFile = `${options.main}`
+    map((project) => {
+      const root = project.root
+      const sources = `${root}/**/*.go`
 
-      return runGoCommand(context, "fmt", [mainFile])
+      return runGoCommand(context, "fmt", [sources])
     }),
   );
 }

--- a/packages/nx-go/src/builders/test/builder.ts
+++ b/packages/nx-go/src/builders/test/builder.ts
@@ -9,10 +9,11 @@ export function runBuilder(
   context: BuilderContext
 ): Observable<BuilderOutput> {
   return from(context.getProjectMetadata(context?.target?.project)).pipe(
-    map(() => {
-      const mainFile = `${options.main}`
+    map((project) => {
+      const root = project.root
+      const sources = `-v ${root}/**/*.go`
 
-      return runGoCommand(context, "test", [mainFile])
+      return runGoCommand(context, "test", [sources])
     }),
   );
 }

